### ISSUE TITLE
No Scrutinizer Runs Today

### ISSUE THREAD

**Issue by oleibman** [PHPOffice/PhpSpreadsheet#3788]
@PowerKiKi @MarkBaker No PRs submitted today have run a Scrutinizer step. Are you aware of any changes that would prevent it?

And, as long as I have your attention, what do you think of PR 3778 (make Php 8.3 not experimental)? I think we're only a couple of weeks away from its official release. Can we merge 3778 after that happens (the nightly run has had no failures/errors for several weeks now), mark Php 8.3 as required (I don't know how to do that), and then make an official release?

**Comment by PowerKiKi** [comment]
I merged https://github.com/PHPOffice/PhpSpreadsheet/pull/3778, where Srcutinizer ran successfully. I suppose something went temporarily wrong on their side ...

**Comment by PowerKiKi** [comment]
Btw, this remind me that I never released the native typing I added in https://github.com/PHPOffice/PhpSpreadsheet/pull/3718. I added a lot of native typing, but all. Especially I did not touch classes/interfaces that are expected to be inherited, eg `IValueBinder`, because this would be a major change. But I think we should do that at some point...

My opinion is to add typing to the remaining classes/interfaces and **release all of this as a 2.0.0**. @MarkBaker, @oleibman would that be OK with you ?
